getting_started.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5682e81-741c-4fbc-bc1f-974a3b93c831",
+   "metadata": {},
+   "source": [
+    "# Getting Started With The Explorer API Client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8049a67b-9477-430b-a9ec-0fea113d7b9e",
+   "metadata": {},
+   "source": [
+    "## Authentication\n",
+    "\n",
+    "The explorer API validates every request it receives by generating a cryptographic digest using the filters and a secret key associated with your account in Altmetric Explorer and comparing it to the value of the `digest` query parameter included with the request.  If the values are the same then a data response will be returned.  If not, then the API will respond with HTTP status `400 Bad Request`.\n",
+    "\n",
+    "This means you need to calculate the digest using the same method as the API server does when you create a request.  The process is quite involved (see the [documentation](https://www.altmetric.com/explorer/documentation/api#authentication) for more details) but the API client handles this part for you as long as you supply your api key and api secret when you create a new `Client` object.\n",
+    "\n",
+    "You can get you api key and secret at https://www.altmetric.com/explorer/settings.\n",
+    "\n",
+    "You can use whatever you like to distribute secrets but, for the purposes of demonstrating how the API works, we are using [python-dotenv](https://pypi.org/project/python-dotenv/) and you must set it up with your own key and secret for the examples to work.\n",
+    "\n",
+    "Simply create a file called `.env` in the same folder as this notebook and add the following lines using your own keys from https://www.altmetric.com/explorer/settings.\n",
```

Comment:
Would saying "Simply create a file called `.env` in the project root directory (the same directory this notebook is located)" be too technical?

Reply:
I agree with the sentiment but is "project root directory" any less technical?

Reply:
I think "project root" is more technical but also probably more familiar to developers so the above covers both audiences. Happy for you to make the final call though!

Reply:
No real opinion on this myself so batter on!

Reply:
Applied your change Alex